src/Runner/Job.php

```diff
@@ -245,4 +245,24 @@ public function getHeaders()
 		return $this->headers;
 	}
 
+
+	/**
+	 * @return string
+	 */
+	private function buildArgumentsList()
+	{
+		$args = [];
+
+		foreach ($this->args as $key => $arg) {
```

Comment:
IHMO current implementation is better, associative array disallows multi values as you said and --$key=$arg is not standard format.